templates/components/common/responsive-img.html

```diff
@@ -0,0 +1,51 @@
+{{!--
+This file implements the lazysizes srcset pattern (as seen on
+https://github.com/aFarkas/lazysizes#modern-transparent-srcset-pattern) for maximum performance on modern browsers
+and reasonable compatibility with legacy browsers.
+
+There are a few important arguments to know about when using this component:
+`lazyload`:
+    - If set to 'lazyload+lqip', will load a low-res image which will be replaced by lazysizes - ensuring that SOME
+    content is shown immediately on page load. Without LQIP, a transparent gif will be used instead, which reduces
+    the number of requests needed to load the page, making it even faster.
+    - If set to 'lazyload', the browser will not load the full-size image at first, instead the lazysizes plugin will
+        load it progressively after page load. This improves the load speed of the page, but it should be disabled for
+        high-priority above-the-fold images.
+    - If set to 'disabled', will not attempt to lazyload the image and instead tell the browser to load it immediately.
+        Use this for high-priority images that are above the fold.
+
+`fallback_size`:
```

Comment:
You should document `lqip_size` and `default_image` as well.

Reply:
Good callout, will add that.